### DISCUSSION TITLE
deploying-asp-net-core-applications-to-kubernetes-part-5-setting-environment-variables-in-a-helm-chart/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#119] [Announcements]
# deploying-asp-net-core-applications-to-kubernetes-part-5-setting-environment-variables-in-a-helm-chart/

In this post I show how you can pass environment variables to your ASP.NET Core apps deployed in Kubernetes using Helm

https://andrewlock.net/deploying-asp-net-core-applications-to-kubernetes-part-5-setting-environment-variables-in-a-helm-chart/

**Comment by IanMoroney** [comment]
Really great article!
It described perfectly some of the challenges I was trying to overcome, and offered a great dynamic solution to the problem.
Ticks all the boxes!

[a comment by an automated account is not shown]

**Comment by codekoenig** [comment]
That's a really excellent article and series because it discusses getting all of this done in a real-world scenario and not some tutorial that leaves you alone when the real work to get this up and running begins.
